Name: Fullscreen Browser
Version: 1.3.5-1
Author: Fritos1406
Description: When scrolled away from the top when the urlbar is not visibible, Will make it enter fullscreen mode

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
@@ -299,6 +299,9 @@
 		
 		this._updateOrientation();
 		
+		// FullScreen Browser
+        this.controller.enableFullScreenMode(true);		
+		
 		// Update any pending web preferences changes
 		AppAssistant.WebPreferences.activate(this._webView.mojo);
 
@@ -1862,7 +1865,7 @@
 		if (!this._addressBar.hasFocus()) {
 			this._addressBar.hide();
 			this.chrome.hide();
-
+		    this.controller.enableFullScreenMode(true);
 			// Only focus this webview widget if this assistant has
 			// focus. All page-assisants can receive 'moved' events
 			// so we have to be careful when explicitly setting the
@@ -1885,7 +1888,7 @@
 		//Mojo.Log.info("-> TOP > 0 SO SHOWING");
 		this._addressBar.show();
 		this.chrome.show();
-
+		this.controller.enableFullScreenMode(false);
 	} else {
 		//Mojo.Log.info("-> TOP == 0 Do Nothing.");
 		// Do nothing.
